src/test/java/kg/jarkyn/cobspecserver/doubles/ClientDouble.java

```diff
@@ -0,0 +1,29 @@
+package kg.jarkyn.cobspecserver.doubles;
+
+import kg.jarkyn.cobspecserver.Client;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class ClientDouble implements Client {
```

Comment:
In the `ServerTest` there's a line `assertEquals("hello", client.getOutputStream().toString())`.
I'd add a convenience method to `client` to have it read something like `assertEquals("hello", client.getResponseContent())` or something similar.